canonical-websites/build.snapcraft.io
# Missing repos info in 'Add repos' dialog

Fixes #715

**DEPENDS ON #827 #838**

This PR builds on code from #827 and #838, and contains commits also from these PRs, so these 2 should be reviewed and merged before reviewing this one.

---

This PR updates Add repos dialog and moves 'missing repos' info out from the popover to panel in the dialog.
<img width="1044" alt="screen shot 2017-06-16 at 14 32 19" src="https://user-images.githubusercontent.com/83575/27227017-f02690bc-52a1-11e7-8fd0-db0e1259adfe.png">
<img width="1040" alt="screen shot 2017-06-16 at 14 32 35" src="https://user-images.githubusercontent.com/83575/27227018-f026fc28-52a1-11e7-8de1-53550efae9c9.png">
![add-repos](https://user-images.githubusercontent.com/83575/27227056-0d2b96da-52a2-11e7-9c91-8358f329be2f.gif)

***QA:***

1. Sign in
2. Go to 'Add repos'
3. Click 'Anything missing?'
4. List of repos should be replaced with a panel giving some info about how to access other repos.
